Releasing.md

```diff
@@ -213,29 +211,21 @@ For the body of the post, just copy this checklist and again replace all occurre
 <!-- /wp:paragraph -->
 
 <!-- wp:heading {"level":3} -->
-<h3>Sync the Release to Development Branches</h3>
+<h3>Merge Release Branches</h3>
 <!-- /wp:heading -->
 
-<!-- wp:paragraph -->
-<p>o If there are any conflicts in the gutenberg PR, merge <code>trunk</code> into it and resolve them.</p>
-<!-- /wp:paragraph -->
-
-<!-- wp:paragraph -->
-<p>o In gutenberg-mobile, create a branch from <code>trunk</code> with a name like <code>merge_release_X.XX.X_to_develop</code> and open PR to <code>develop</code>. If there are any merge conflicts, merge <code>develop</code> into the PR and resolve them.</p>
-<!-- /wp:paragraph -->
-
 <!-- wp:paragraph -->
 <p>o Merge the gutenberg PR to <code>trunk</code>.</p>
 <!-- /wp:paragraph -->
 
 <!-- wp:paragraph -->
-<p>o Update the <code>gutenberg</code> reference in the gutenberg-mobile <code>merge_release_X.XX.X_to_develop</code> PR so it points to merge commit in gutenberg <code>trunk</code> for the gutenberg PR merged in the previous task.</p>
-<!-- /wp:paragraph -->
+<p>o Update the gutenberg reference on the gutenberg-mobile release branch to point to the Gutenberg PR merge commit</p>
 
 <!-- wp:paragraph -->
-<p>o Merge the gutenberg-mobile PR to <code>develop</code>.</p>
+<p>o Merge the <strong>gutenberg-mobile</strong> PR to <code>trunk</code>. Use "Create a merge commit" option when merging to avoid lossing any commit history from the release branch.</p>
```

Comment:
```suggestion
<p>o Merge the <strong>gutenberg-mobile</strong> PR to <code>trunk</code>. Use "Create a merge commit" option when merging to avoid losing any commit history from the release branch.</p>
```